markdown/ml/ml3_logistic_regression/index.md

```diff
@@ -4,26 +4,51 @@ title: "NN3 - Logistische Regression"
 menuTitle: "NN3 - Logistische Regression"
 author: "Canan Yıldız (Türkisch-Deutsche Universität)"
 weight: 3
-outcomes: false
+outcomes: true
 # readings:
 #   - key: "Russell2020"
 #     comment: "Kapitel 2 und 3"
 #   - key: "Ertel2017"
 # quizzes:
 #   - link: XYZ
 #     name: "Testquizz (URL from 'Invite more Players')"
-# assignments:
-#   - topic: blatt01
-#   - topic: blatt02
-# youtube:
-#   - id: XYZ (ID)
-#   - id: XYZ (ID)
+assignments:
+    - topic: sheet07
+youtube:
+    - id: GpJmjrqA5RY
+    - id: z-jFZeNWMRc
+    - id: ruuCKupOhCE
+    - id: kPAZsr-r1LA
 ---
 
+## Kurze Übersicht
 
-<div style="text-align:center;">
-Inhalt befindet sich im Aufbau<br>
-und wird rechtzeitig bereitgetellt.
-</div>
+### Formalisierung
+*   Ausgabe $y$ ist reelle Zahl aus dem stetigen Bereich $(0,1)$
+*   Die **Hypothesenfunktion** ist:
+    $$ h(\mathbf{x}) = \sigma (\mathbf{w}^T\mathbf{x}) = \sigma (w_0 + w_1x_1 + w_2x_2 + \ldots + w_nx_n) $$
+
+*   Der **Kreuzentropie Verlust** (engl. Cross-Entropy) für einen Datenpunkt $ \mathbf{x} $: 
+    $$ \mathcal{L}(a, y) =  - y  \log(a) - (1-y)  \log(1-a)\tag{3} $$
```

Comment:
@cyildiz Mir ist grad aufgefallen, dass Du die Formeln mit Nummern versiehst, aber nicht bei (1) anfängst und auch Sprünge in der Nummerierung hast. Ist das so gewollt oder soll das noch geändert werden?

Reply:
@cagix das hatte ich gar nicht bemerkt. 
Ich habe die Formeln von meinem Jupyter Notebook kopiert, dort hatte ich die `\tag` s drin. 
sieht aber nicht schlecht aus, habe sie drin gelassen und korrigiert.
danke für den hinweis!
jetzt kann gemergt werden denke ich.

Reply:
sehr cool! ich kannte das `\tags{}` noch gar nicht. in latex lasse ich das immer automatisch nummerieren, aber für html hat das immer nicht funktioniert.